ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -136,12 +136,6 @@ Verify RHODS Installation
     ...    label_selector=app.kubernetes.io/name=data-science-pipelines-operator    timeout=400s
   END
 
-  ${modelregistry} =    Is Component Enabled    modelregistry    ${DSC_NAME}
```

Comment:
Why are we changing model registry for trustyai?

Reply:
I moved the model registry section after the trustyai one, because model registry reconciliation happens after the trustyai one, so just to follow the same order to check that the operator does